projects/components/src/summary-list/summary-list.component.ts

```diff
@@ -35,7 +35,7 @@ export class SummaryListComponent {
   public items?: SummaryItem[] = [];
 
   public getFormattedLabel(item: SummaryItem): string {
-    return startCase(item.label.trim().replace('-', ' ').replace('_', ' ').toLowerCase());
+    return startCase(item.label.trim().toLowerCase());
```

Comment:
trim is definitely still taken care of by startCase, the most intelligent of all cases. lower case may not be though.